src/dos/program_mount.cpp

```diff
@@ -59,7 +59,8 @@ void MOUNT::Move_Z(char new_z)
 		/* Update environment */
 		std::string line = "";
 		std::string tempenv = {new_drive_z, ':', '\\'};
-		if (first_shell->GetEnvStr("PATH",line)) {
+		if (auto result{first_shell->GetEnvStr("PATH")}) {
+			line = std::move(*result);
```

Comment:
Oh, man, just copy the string with `=`, let the compiler deal with it 😄 `std::move` should be reserved for performance intensive code when there is tangible difference.